### ISSUE TITLE
Fix pyflakes for news

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6991]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)| @adiroiban reported|
|-|-|
|Trac ID|trac#6991|
|Type|enhancement|
|Created|2014-02-27 12:17:41Z|
|Branch|https://github.com/twisted/twisted/tree/pyflakes-news-6991|

Attachments:

* [6991-1.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/792/792eed93e471be02b2efbef737e7c795c655a564/879baca51db673f444956c7bd1c2d1bb1d39c275.diff) (478 bytes) - added by adiroiban on 2014-02-27 12:18:49Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__6991 6991
type__enhancement enhancement
reporter__adiroiban adiroiban
priority__normal normal
milestone__ 
branch__branches_pyflakes_news_6991 branches/pyflakes-news-6991
branch_author__hawkowl hawkowl
status__closed closed
resolution__fixed fixed
component__news news
keywords__ 
time__1393503461000000 1393503461000000
changetime__1393508586000000 1393508586000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_1"></a>|@adiroiban commented|
|-|-|

Only unused imports in a test.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

(In [41761]) Branching to pyflakes-news-6991.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

(In [41763]) Apply 6991-1.diff

refs [#6991](https://github.com/twisted/twisted/issues/6991)

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl commented|
|-|-|

Looks good to me, merging.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl set status to `closed`|
|-|-|

(In [41767]) Merge pyflakes-news-6991

Author: adiroiban
Reviewer: hawkowl
Fixes: [#6991](https://github.com/twisted/twisted/issues/6991)

Removes some unneeded imports, making Twisted News "pyflakes clean".
